docs/user-guide/overview.md

```diff
@@ -1,75 +1,71 @@
-- **Range Queries:** GreptimeDB supports [range queries](/user-guide/query-data/sql.md#aggregate-data-by-time-window) to evaluate [expressions](/reference/sql/functions/overview.md) over time, providing insights into metric trends. You can also [continuously aggregate](/user-guide/flow-computation/overview.md) data for further analysis.
-- **SQL and Multiple Protocols:** GreptimeDB uses SQL as the main query language and supports [multiple protocols](/user-guide/protocols/overview.md), which greatly reduces the learning curve and development cost. You can easily migrate from Prometheus or [Influxdb to GreptimeDB](/user-guide/migrate-to-greptimedb/migrate-from-influxdb.md), or just start with GreptimeDB.
-- **JOIN Operations:** The data model of GreptimeDB's time series tables enables it to support [JOIN operations](/reference/sql/join.md) on metrics and logs.
-
-Having understood these features, you can now go directly to exploring the features that interest you, or continue reading the next step in the sequence.
-
-## Next steps
-
-* [Concepts](./concepts/overview.md)
-* [Migrate to GreptimeDB](./migrate-to-greptimedb/migrate-from-influxdb.md)
-* [Ingest Data](./ingest-data/overview.md)
-* [Query Data](./query-data/overview.md)
-* [Manage Data](./manage-data/overview.md)
-* [Integrations](./integrations/overview.md)
-* [Protocols](./protocols/overview.md)
-* [Continuous Aggregation](./flow-computation/overview.md)
-* [Operations](./administration/overview.md)
+
+## Understanding GreptimeDB Concepts
+
+Before diving into GreptimeDB,
+it is recommended to familiarize yourself with its data model, key concepts, and features.
+For a comprehensive overview, 
+refer to the [Concepts Documentation](./concepts/overview.md).
+
+## Ingesting Data Based on Your Use Case
+
+GreptimeDB supports [multiple protocols](./protocols/overview.md) and [various integration tools](./integrations/overview.md) to simplify data ingestion tailored to your requirements.
+
+### For Observability Metrics
+
+If you plan to use GreptimeDB as metrics storage for observability purposes,
+see the [Observability Documentation](./ingest-data/for-observability/overview.md).
+It explains how to ingest data using tools like Vector, Kafka, Prometheus, and the InfluxDB line protocol.
+
+### For IoT and Edge Computing
+
+For IoT and Edge Computing scenarios,
+the [IoT Documentation](./ingest-data/for-iot/overview.md) provides comprehensive guidance on ingesting data from diverse sources.
+
+### For Log Storage
+
+To use GreptimeDB as a log storage solution,
+refer to the [Logs Documentation](./logs/overview.md).
+It details how to ingest pattern text logs using pipelines.
+
+### Accelerating Queries with Indexes
+
+Indexes such as inverted indexes, skipping indexes, and full-text indexes can significantly enhance query performance.
+Learn more about effectively using these indexes in the [Data Index Documentation](./manage-data/data-index.md).
+
+## Querying Data for Insights
```

Comment:
This section must be above `Accelerating Queries with Indexes`.

You should learn how to query data, then cccelerating them.